### PR TITLE
Sage

### PR DESCRIPTION
**Pull request by CamHardy**
Sage

### CODE REVIEW

**Comment by zpalmtree** [comment]
Why are you adding a package-lock.json this uses yarn

File: package.json [hunk 1]
```diff
@@ -1,9 +1,13 @@
 {
   "name": "calculate-4-4-profit",
+  "author": "Cam Hardy",
```
Comment (zpalmtree) [inline package.json:3]: really bro, there is a contributors field

**Comment by zpalmtree** [comment]
maybe just change the readme next time fuck it i'll do it myself

File: package.json (hunk 1 above)
Comment (CamHardy) [reply package.json:3]: Well darn you weren't supposed to see that

**Pull request closed without merging**